Formula/osslsigncode.rb

```diff
@@ -3,6 +3,7 @@ class Osslsigncode < Formula
   homepage "https://sourceforge.net/projects/osslsigncode/"
   url "https://downloads.sourceforge.net/project/osslsigncode/osslsigncode/osslsigncode-1.7.1.tar.gz"
   sha256 "f9a8cdb38b9c309326764ebc937cba1523a3a751a7ab05df3ecc99d18ae466c9"
+  head "http://git.code.sf.net/p/osslsigncode/osslsigncode"
```

Comment:
As far as I can see, there's no pre-generated `configure` file in the upstream git repo, so this won't work by itself. You'll get a `Failed to execute ./configure` message or similar.